Add the manual download and install for Windows section (#16913)
dotnet/docs: docs/core/install/runtime.md

>
> Also, add `export DOTNET_ROOT=$HOME/dotnet` to the end of the file.

> [!TIP]

> > This approach lets you install multiple different versions into separate locations and choose explicitly which one to use by which application.

Reviewer: 1. I would remove the tip here and have plain text. This isn't really a tip as it is explaining something.
2. I don't see how this enables choice of explicitly choosing which to use. Maybe you can be more clear such as set the environment variable per app?

> [!NOTE]
> The command above installs the ASP.NET Core runtime for maximum compatability. The ASP.NET Core runtime also includes the standard .NET Core runtime.

## Download and manually install

> First [download](#all-net-core-downloads) a .NET Core binary release. Then create a directory to install to, for example `%USERPROFILE%\dotnet`. And finally extract the downloaded zip file into that directory.

Reviewer 2: Borrow more from the Linux page to make it smoother?

```suggestion
To extract the runtime and make the .NET Core CLI commands available at the terminal, first [download](#all-net-core-downloads) a .NET Core binary release. Then, create a directory to install to, for example `%USERPROFILE%\dotnet`. Finally, extract the downloaded zip file into that directory.
```

> .NET Core installed in this way will not be picked up by default, you have to explicitly choose to use it. To do so, change the environment variables with which an application is started:

Reviewer 2: A few things:
* I think "picked up" could be clarified. Edited arrangement a bit for active voice and removed comma splice:
  ```suggestion
  By default, .NET Core CLI commands and apps will not use .NET Core installed in this way. You have to explicitly choose to use it. To do so, change the environment variables with which an application is started:
  ```
* I think this should clarify that these are console (not powershell) commands. Or can it offer a switch to show either syntax?
* Seems it should also mention that it will only take effect in the terminal they're typed into. The Linux side is more detailed about env vars... how to set them permanently, for example. Might need similar detail on Windows?

Author: On Windows it's very unlikely users will want to install into a custom location and set that as the "global" install (unlike Linux, where for some distros that's the only way to do it right now). So I intentionally omitted the part about global setup.

Also in 3.0 the global registration can now be done via registry/conf files - which is a different type of change. Not sure it should be mentioned here though - as that's mostly intended for installers (and advanced usage).

```
set DOTNET_ROOT=%USERPROFILE%\dotnet
set PATH=%USERPROFILE%\dotnet;%PATH%
```

> This approach lets you install multiple different versions into separate locations and explicitly choose which application should use which install location by running the application with its own set of environment variable values. Even with the application started under this environment .NET Core will still consider the default global install location (typically `C:\Program Files\dotnet`) which is used by the installers when selecting the best framework for running the application. You can instruct the runtime to only use the custom install location by setting this environment variable as well:

Reviewer 2: Seems like `Even with the application started under this environment...` should start a new paragraph.

> [!NOTE]
> The command above installs the ASP.NET Core runtime for maximum compatability. The ASP.NET Core runtime also includes the standard .NET Core runtime.

> ## Download and manual install

Reviewer: To match the linux section

```suggestion
## Download and manually install
```

First [download](#all-net-core-downloads) a .NET Core binary release. Then create a directory to install to, for example `%USERPROFILE%\dotnet`. And finally extract the downloaded zip file into that directory.

.NET Core installed in this way will not be picked up by default, you have to explicitly choose to use it. To do so, change the environment variables with which an application is started:

> ```

Reviewer: You should use the code language `console` for these

set DOTNET_ROOT=%USERPROFILE%\dotnet
set PATH=%USERPROFILE%\dotnet;%PATH%
```

> [!TIP] 

> > This approach lets you install multiple different versions into separate locations and choose explicitly which one to use by which application. Even with the application started under this environment, .NET Core will still consider the default global install location (typically `C:\Program Files\dotnet`) which is used by the installers. To disable this and ask the runtime to only consider the custom install location also modify this environment variable:

Reviewer: - Is this really *"to disable this and ask the runtime to..."* or is it *"to disable this and have the runtime..."*?
- Maybe adjust the intro to be explicit about that the "the environment variable change lets you explicitly set where dotnet core is located" and point at that if you change these variables for an application, only that app is affected.

> [!NOTE]
> The command above installs the ASP.NET Core runtime for maximum compatability. The ASP.NET Core runtime also includes the standard .NET Core runtime.

## Download and manually install

To extract the runtime and make the .NET Core CLI commands available at the terminal, first [download](#all-net-core-downloads) a .NET Core binary release. Then, create a directory to install to, for example `%USERPROFILE%\dotnet`. Finally, extract the downloaded zip file into that directory.

By default, .NET Core CLI commands and apps will not use .NET Core installed in this way. You have to explicitly choose to use it. To do so, change the environment variables with which an application is started:

> ```

Reviewer 2: ```suggestion
```console
```

set DOTNET_ROOT=%USERPROFILE%\dotnet
set PATH=%USERPROFILE%\dotnet;%PATH%
```

> This approach lets you install multiple different versions into separate locations and explicitly choose which application should use which install location by running the application with its own set of environment variable values.

Reviewer 2: ```suggestion
This approach lets you install multiple versions into separate locations, then explicitly choose which install location an application should use by running the application with environment variables pointing at that location.
```

Reviewer: I like this change

> Even with the application started under this environment .NET Core will still consider the default global install location (typically `C:\Program Files\dotnet`) which is used by the installers when selecting the best framework for running the application. You can instruct the runtime to only use the custom install location by setting this environment variable as well:

Reviewer 2: ```suggestion
Even when these environment variables are set, .NET Core still considers the default global install location when selecting the best framework for running the application. The default is typically `C:\Program Files\dotnet`, which the installers use. You can instruct the runtime to only use the custom install location by setting this environment variable as well:
```

Reviewer: I like this suggestion
